dist/images/Dockerfile.dpdk1911

```diff
@@ -0,0 +1,72 @@
+# NOTE: At time of writing, CentOS 8 does not have dpdk-devel v19.11 in the Yum repo.
+# This package is required to build the OVS-DPDK RPMs and is available in Fedora 32.
+FROM fedora:32
+
+ENV RPM_ARCH=x86_64
+ENV ARCH=amd64
+
+ENV OVN_VERSION=20.03
+ENV OVS_VERSION=2.13.0
+
+ENV DPDK_VERSION=19.11.1
+ENV DPDK_TARGET=x86_64-native-linuxapp-gcc
+
+ENV OVN_DIR=/usr/src/ovn
+ENV OVS_DIR=/usr/src/openvswitch-${OVS_VERSION}
+ENV DPDK_DIR=/usr/src/dpdk-stable-${DPDK_VERSION}
+
+ENV PATH=${PATH}:/usr/share/openvswitch/scripts
+ENV PATH=${PATH}:/usr/share/ovn/scripts/
+
+RUN dnf install -y wget xz git make numactl-devel diffutils dnf-plugins-core dpdk-devel libcap-ng-devel libpcap-devel nc iptables ipset hostname && dnf clean all
+
+# NOTE: Fedora 32 comes with gcc version 10. This caused LD linker issues during DPDK build.
+# Install gcc version 9 and its dependencies.
+RUN mkdir /rpms && cd /rpms
+RUN wget https://kojipkgs.fedoraproject.org//packages/gcc/9.2.1/1.fc32.3/x86_64/cpp-9.2.1-1.fc32.3.x86_64.rpm \
+         https://kojipkgs.fedoraproject.org//packages/gcc/9.2.1/1.fc32.3/x86_64/libgomp-9.2.1-1.fc32.3.x86_64.rpm \
+         https://kojipkgs.fedoraproject.org//packages/gcc/9.2.1/1.fc32.3/x86_64/gcc-9.2.1-1.fc32.3.x86_64.rpm
+RUN dnf install -y cpp-9.2.1-1.fc32.3.x86_64.rpm libgomp-9.2.1-1.fc32.3.x86_64.rpm gcc-9.2.1-1.fc32.3.x86_64.rpm
+RUN rm -f cpp-9.2.1-1.fc32.3.x86_64.rpm libgomp-9.2.1-1.fc32.3.x86_64.rpm gcc-9.2.1-1.fc32.3.x86_64.rpm
```

Comment:
The wget/install/rm steps should in one RUN step to reduce the image size

Reply:
Cool. I'll try this and I'll also see if I can do anything else to reduce image size.